roles/palava-signaltower/templates/signaltower.service.j2

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Palava Signal Tower
+After=network.target
+
+[Service]
+Type=simple
+User=plv
+Group=plv
+WorkingDirectory={{ palava_signaltower_install_dir }}/
+ExecStart=/usr/bin/elixir --sname signaltower -S mix run --no-halt
```

Comment:
Just like in the current systemd unit file I would like to keep the starting and stopping over the release file (ExecStart/Stop=$WorkingDirectory/_build/prod/rel/production/bin/production daemon/stop). This has at least the advantage that we can also over the release file attach a live console to the tower to debug problems)

Reply:
What I really don't like about this approach is that you start a seperate background process with the "daemon start". I actually prefer "simple" systemd services - meaning a service started in foreground managed by systemd. That just makes everything a lot easier to manage and less error prone.

Reply:
...which is resolved now that you chose start over daemon, right?

Reply:
I would say so.